### PR TITLE
Report "CAM error" when SensorFailure occurs

### PR DESCRIPTION
**Pull request by bmerry**
These errors are invariably due to problems in katportal/katstore, and
this will hopefully direct operators to the right people for debugging.

### CODE REVIEW

File: katsdpcontroller/sdpcontroller.py [hunk 1]
```diff
@@ -1435,10 +1435,14 @@ def remove_product(product):
             product_config.validate(config_dict)
             config_dict = await product_config.update_from_sensors(config_dict)
             config_dict = product_config.normalise(config_dict)
-        except (ValueError, product_config.SensorFailure, jsonschema.ValidationError) as error:
+        except product_config.SensorFailure as error:
+            retmsg = "CAM error: {}".format(error)
```
Comment (sratcliffe) [inline katsdpcontroller/sdpcontroller.py]: I know I asked for this, but are we being too blunt in calling this a CAM error :) Apart from katstore being completely unavailable or timeouts on sensor requests, what else could trigger SensorFailure ?

Comment (bmerry) [reply katsdpcontroller/sdpcontroller.py]: Easiest answer to that is to search for it in product_config.py. I guess some of the cases (like the sensor not existing) could be due to CBF as well.

Comment (sratcliffe) [reply katsdpcontroller/sdpcontroller.py]: OK - how about "Sensor error (possible CAM problem): {}" ?

Comment (martinslabber) [reply katsdpcontroller/sdpcontroller.py]: Maybe be a bit more helpful and say: "Could not get sensor DATA from CAM:", DATA is not clear but sure you can substitute.

Comment (sratcliffe) [reply katsdpcontroller/sdpcontroller.py]: Final bikeshed attempt: "Error retrieving sensor data from CAM: {}"

Comment (bmerry) [reply katsdpcontroller/sdpcontroller.py]: Done

**Pull request merged**